src/lib/Containers/Conversation.tsx

```diff
@@ -156,6 +158,7 @@ export class Conversation extends React.Component<Props, State> {
           </Header>
           {!this.state.isConnected && <ConnectivityBanner />}
           <Messages
+            ref={messages => (this.messages = messages)}
```

Comment:
Use `componentRef` instead so you don’t need to use `this.messages.refs.component` above